redis-pq-demo/docker-compose.yml

```diff
@@ -0,0 +1,95 @@
+version: '3'
+services:
+  caddy:
```

Comment:
no `ngrok`? running this as is will not work with your other steps. other agents (ie `playground`) will have to join this network to use this. i think you should have `ngrok` on by default and notes about how to turn it off if needed.

Reply:
reinstated.